packages/devtools/lib/src/ui/ui_utils.dart

```diff
@@ -23,6 +24,26 @@ const int defaultSplitterWidth = 10;
 const String runInProfileModeDocsUrl =
     'https://flutter.dev/docs/testing/ui-performance#run-in-profile-mode';
 
+// GA dimensions:
+String userAppType = ''; // dimension1
+String userBuildType = ''; // dimension2
+String userPlatformType = ''; // dimension3
+
+void computeApplicationState(
+  bool isAnyFlutterApp,
+  bool isFlutter,
+  bool isWebApp,
+  bool isProfile,
+  bool isAndroid,
```

Comment:
Can we replace isAndroid with the platform name? We have web, android, ios, fuchsia, etc.

Reply:
I use these to determine how to pull the platform information (which code I just added) in (e.g., if isFlutter then I can use Platform to get Fuchsia, Android, iOS, etc.